src/vm/memory/memory.zig

```diff
@@ -207,6 +207,69 @@ pub const Memory = struct {
     }
 };
 
+// Utility function to help set up memory for tests
+//
+// # Arguments
+// - `memory` - memory to be set
+// - `vals` - complile time structure with heterogenous types
+fn memoryInner(memory: *Memory, comptime vals: anytype) !void {
+    inline for (vals) |row| {
+        const firstCol = row[0];
+        const relo = Relocatable.new(firstCol[0], firstCol[1]);
+        // Check number of inputs in row
+        if (row[1].len == 1) {
+            const nextCol = row[1];
+            const maybeFromFelt = MaybeRelocatable{ .felt = Felt252.fromInteger(nextCol[0]) };
+            try memory.set(relo, maybeFromFelt);
```

Comment:
Here, it's a preference that I have to make the lines compact, it's whatever you prefer, I think that the three lines above can be reduced to:

```
try memory.set(relo, .{ .felt = Felt252.fromInteger(row[1][0]) });
```

Reply:
Yes I like this is much better not sure I was able to access row[1][0] like that but since the change to a const row[1] outside of if statement its no longer an concern.